### ISSUE TITLE
SVG Zoom broken

### ISSUE THREAD

**Issue by tyteen4a03** [haxiomic/firefox-multi-touch-zoom#49]
On large SVG files ([this one for example](https://www.gleisplanweb.eu/Maps/Stuttgart.svg)), once zoomed in it's not possible to scroll to the other areas of the file.

I am using Firefox 75 on macOS 10.15.4, zooming with my trackpad.

**Comment by haxiomic** [comment]
Yep same for me for FF 74 on macOS, I found it works however if I use the new built-in zoom feature which is hidden behind a config flag. Here's how to enable it:
- Remove my extension from your browser
- Go to `about:config` in the URL bar
- Change the setting `apz.allow_zooming` from *false* to *true*

Restart Firefox
